apps/mail/components/ui/nav-main.tsx

```
@@ -218,9 +211,9 @@ function NavItem(item: NavItemProps & { href: string }) {
     >
       {item.icon && <item.icon ref={!item.isBackButton ? iconRef : undefined} className="" />}
       <p className="mt-0.5 text-[13px]">{item.title}</p>
-      {stats && stats.find((stat) => stat.folder === item.title.toLowerCase()) && (
+      {stats && stats.find((stat) => stat.label.toLowerCase() === item.title.toLowerCase()) && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)